cmd/build/helmify/replacements.go

```diff
@@ -114,6 +114,80 @@ var replacements = map[string]string{
 
+	"- HELMSUBST_CONTROLLER_MANAGER_OPERATIONS": `
+        {{- if not (hasKey .Values.controllerManager "disableWebhookOperation") }}
+        - --operation=webhook
+        {{- else if not .Values.controllerManager.disableWebhookOperation }}
+        - --operation=webhook
+        {{- end }}
+        {{- if hasKey .Values.controllerManager "disableGenerateOperation" }}
+        {{- if not .Values.controllerManager.disableGenerateOperation }}
+        - --operation=generate
+        {{- end }}
+        {{- end }}
+        {{- if hasKey .Values.controllerManager "disableMutationWebhookOperation" }}
+        {{- if not .Values.controllerManager.disableMutationWebhookOperation }}
+        - --operation=mutation-webhook
+        {{- end }}
+        {{- end }}
+        {{- if hasKey .Values.controllerManager "disableAuditOperation" }}
+        {{- if not .Values.controllerManager.disableAuditOperation }}
+        - --operation=audit
+        {{- end }}
+        {{- end }}
+        {{- if hasKey .Values.controllerManager "disableStatusOperation" }}
+        {{- if not .Values.controllerManager.disableStatusOperation }}
+        - --operation=status
+        {{- end }}
+        {{- end }}
+        {{- if hasKey .Values.controllerManager "disableMutationStatusOperation" }}
+        {{- if not .Values.controllerManager.disableMutationStatusOperation }}
+        - --operation=mutation-status
+        {{- end }}
+        {{- end }}
+        {{- if hasKey .Values.controllerManager "disableMutationControllerOperation" }}
+        {{- if not .Values.controllerManager.disableMutationControllerOperation }}
+        - --operation=mutation-controller
+        {{- end }}
+        {{- end }}`,
+
+	"- HELMSUBST_AUDIT_OPERATIONS": `
+        {{- if not (hasKey .Values.audit "disableWebhookOperation") }}
+        - --operation=webhook
+        {{- else if not .Values.audit.disableWebhookOperation }}
+        - --operation=webhook
+        {{- end }}
+        {{- if hasKey .Values.audit "disableGenerateOperation" }}
+        {{- if not .Values.audit.disableGenerateOperation }}
+        - --operation=generate
+        {{- end }}
+        {{- end }}
+        {{- if hasKey .Values.audit "disableMutationWebhookOperation" }}
+        {{- if not .Values.audit.disableMutationWebhookOperation }}
+        - --operation=mutation-webhook
+        {{- end }}
+        {{- end }}
+        {{- if hasKey .Values.audit "disableAuditOperation" }}
+        {{- if not .Values.audit.disableAuditOperation }}
+        - --operation=audit
+        {{- end }}
+        {{- end }}
+        {{- if hasKey .Values.audit "disableStatusOperation" }}
+        {{- if not .Values.audit.disableStatusOperation }}
+        - --operation=status
+        {{- end }}
+        {{- end }}
+        {{- if hasKey .Values.audit "disableMutationStatusOperation" }}
+        {{- if not .Values.audit.disableMutationStatusOperation }}
+        - --operation=mutation-status
```

Comment:
We already have a variable called `disableMutation` for this, so let's not add another variable for the same thing. I need to think about how to depricate other one without breaking backwardCompatibility.

Reply:
Let's remove this variable since we have another one for this.